wiki/en/Installation-for-Windows.md

```diff
@@ -20,11 +20,12 @@ Upgrading? You may want to [back up your configuration](Software-Manual#backing-
 1. **Run Jamulus**. Now you should be able to use Jamulus just like any other application.
 1. **Set up your sound card**. When you're done, you need to set up your audio hardware. And if you are using ASIO4ALL, have a look at how to set that up below.
 
+_Please note that you are not permitted to redistribute this binary without acquiring a [licence agreement from Steinberg](https://www.steinberg.net/developers/)._
+
 ***
 
-## ASIO
+## Jamulus uses [ASIO](https://en.wikipedia.org/wiki/Audio_Stream_Input/Output) 
```

Comment:
Have you read the Wikipedia page?  It's dire...

Reply:
Like I say, I'm slightly reluctant to link to Steinberg, but maybe I'm just paranoid.

Reply:
I'm only reluctant because it'll mean we get link breakage (do we run a link check as part of the doc build?).

Reply:
> do we run a link check as part of the doc build?

No. I gave up on that a while ago.

Reply:
I've put the link back in to ASIO4all now anyway.

Reply:
> I'm only reluctant because it'll mean we get link breakage (do we run a link check as part of the doc build?).

We do manually.